src/resolver.ts

```diff
@@ -11,35 +16,60 @@ async function get(url: string): Promise<any> {
   return res.json()
 }
 
-export function getResolver() {
+export function getResolver(): Record<string, DIDResolver> {
   async function resolve(
     did: string,
     parsed: ParsedDID
-  ): Promise<DIDDocument | null> {
-    let path = parsed.id + DOC_PATH
+  ): Promise<DIDResolutionResult> {
+    let err = null
+    let path = decodeURIComponent(parsed.id) + DOC_PATH
```

Comment:
Is that correct? we would allow someing like `did:web:veramo%2Eio`? -> urldecoded(veramo.io)

Reply:
Yes, this is done to support w3c-ccg/did-method-web#7

Without this change, this DID method is not really compliant since some characters in the hostname or path might not be valid `id-char`.

Reply:
Ok, than that's fine for me.